src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="/">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
```

Comment:
The wrong path in link, you should use "#"

Reply:
agree